processors/events_dashboard_processor.py

```diff
@@ -0,0 +1,179 @@
+import os
+import argparse
+import boto3
+import json
+import logging
+from fastavro import reader
+from datetime import datetime, timezone
+from urllib.parse import unquote
+import hashlib
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from .common import (
+    ProcessorException,
+    processor_runner,
+    commit_ignoring_unique_violations,
+)
+from raise_data.dashboard.schema import (
+    ContentLoadedEvent,
+    PsetProblemAttemptedEvent,
+    InputSubmittedEvent,
+)
+
+
+logging.basicConfig(level=logging.INFO)
+
+# We setup the sqlalchemy resources at global scope and so read some expected
+# environment variables here. If they end up being missing, we'll fail later
+# anyways.
+
+pg_server = os.getenv("POSTGRES_SERVER", "")
+pg_db = os.getenv("POSTGRES_DB", "")
+pg_user = os.getenv("POSTGRES_USER", "")
+pg_password = os.getenv("POSTGRES_PASSWORD", "")
+sqlalchemy_url = f"postgresql://{pg_user}:{pg_password}@{pg_server}/{pg_db}"
+
+engine = create_engine(sqlalchemy_url)
+session_factory = sessionmaker(engine)
+
+
+def get_config():
+    try:
+        return {
+            "consumer_queue": os.environ["SQS_QUEUE"],
+            "poll_interval_mins": int(os.environ["POLL_INTERVAL_MINS"]),
+            "event_type": os.environ["EVENT_TYPE"],
+            "postgres_server": os.environ["POSTGRES_SERVER"],
+            "postgres_db": os.environ["POSTGRES_DB"],
+            "postgres_user": os.environ["POSTGRES_USER"],
+            "postgres_password": os.environ["POSTGRES_PASSWORD"],
+        }
+    except KeyError as e:  # pragma: no cover
+        raise ProcessorException(f"Missing expected environment variable: {e}")
+
+
+def timestamp_utc_conversion(millisecond_timestamp):
+    # Remove decimal from timestamp_seconds via int()
+    timestamp_seconds = int(millisecond_timestamp / 1000)
```

Comment:
There is not a functional reason. I recall seeing time stamps without the float so I removed it to match the pattern I recalled.
Adding back the float will result in timestamps as shown below. 
![image](https://github.com/openstax/raise-data/assets/37000585/693e0814-cd5b-4241-9350-bbd881607d21)

Reply:
Yeah, I think that's fine / desirable in this case since it includes the millisecond granularity. I mean probably we're only going to ever aggregate by day or hour or something, but better to have it and not need it than need it and not have it.